pkgs/top-level/all-packages.nix

```diff
@@ -5549,7 +5549,13 @@ in
   # These are for compatibility and should not be used inside Nixpkgs.
   pythonFull = python.override{x11Support=true;};
   python2Full = python2.override{x11Support=true;};
+  python26Full = python26.override{x11Support=true;};
```

Comment:
the python26 expression doesn't take x11Support, since I never modified that expression.